App/src/main/webapp/index.html

```diff
@@ -11,5 +11,14 @@
 </head>
 <body>
     <p>Hello :)</p>
+    <h2>About Us</h2>
+    <p>
+        &emsp;The goal of the app is to direct those who are either homeless 
```

Comment:
I noticed this in another file (Jacinda's, perhaps?) that I was reviewing earlier - where are these  "\&emsp\;"s coming from?

Reply:
Those put spaces in the display of the about section but I'll delete them.